### PR TITLE
drivers: usb: device: b9x: enabled IRQ for USP_EP

### PR DESCRIPTION
**Pull request by UR6LAL**
enabled IRQ for USP_EP for Telink B92 platform

### CODE REVIEW

**Comment by andriy-bilynskyy** [comment]
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| hal_telink | https://github.com/telink-semi/hal_telink/commit/c01672a589740d85ff977c331e9f58659caccc80 | https://github.com/telink-semi/hal_telink/commit/b3095cf680881317d58c59fe0aa893cb327763ad (develop) | [telink-semi/hal_telink@c01672a5..b3095cf6](https://github.com/telink-semi/hal_telink/compare/c01672a589740d85ff977c331e9f58659caccc80..b3095cf680881317d58c59fe0aa893cb327763ad) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

File: drivers/usb/device/usb_dc_b9x.c [hunk 1]
```diff
@@ -878,8 +878,12 @@ static int usb_irq_init(void)
 
 	usbhw_enable_manual_interrupt(FLD_CTRL_EP_AUTO_CFG | FLD_CTRL_EP_AUTO_DESC |
 				FLD_CTRL_EP_AUTO_FEAT | FLD_CTRL_EP_AUTO_STD);
+	usbhw_set_eps_irq_mask(FLD_USB_EDP5_IRQ | FLD_USB_EDP6_IRQ);
+
 	core_interrupt_enable();
```
Comment (andriy-bilynskyy) [inline drivers/usb/device/usb_dc_b9x.c]: Do we really need to enable global ISRs. It's driver code so kernel is running in that case ISR's are already enabled. Please check all same entries in this file.

**Review by andriy-bilynskyy — CHANGES_REQUESTED** [review]

File: drivers/usb/device/usb_dc_b9x.c (hunk 1 above)
Comment (mishadesh) [reply drivers/usb/device/usb_dc_b9x.c]: @andriy-bilynskyy 
Do you mean line 883?

Comment (UR6LAL) [reply drivers/usb/device/usb_dc_b9x.c]: @andriy-bilynskyy I removed core_interrupt_enable(), it works ok

**Comment by andriy-bilynskyy** [comment]
Please update manifest file with SHA: b3095cf680881317d58c59fe0aa893cb327763ad

**Review by andriy-bilynskyy — APPROVED** [review]

**Pull request merged**